proxy-node-gateway/src/main/java/uk/gov/ida/notification/resources/EidasAuthnRequestResource.java

```diff
@@ -96,7 +98,7 @@ private View handleAuthnRequest(String encodedEidasAuthnRequest, String eidasRel
         ProxyNodeLogger.addContext(ProxyNodeMDCKey.HUB_REQUEST_ID, vspResponse.getRequestId());
         ProxyNodeLogger.addContext(ProxyNodeMDCKey.HUB_URL, vspResponse.getSsoLocation().toString());
         ProxyNodeLogger.info("Authn requests received from ESP and VSP");
-
+        MetricsUtils.REQUESTS_SUCCESSFUL.inc();
```

Comment:
I don't know if this is a sensible complaint, but it feels like there is still some opportunity for the request to fail, after this counter has been incremented.

Perhaps we could put the value from `buildSamlFormView` into a variable, and increment the counter immediately before we `return`?

Reply:
Good catch, I've pushed to inc() the counter at last possible point